### PR TITLE
Taking dependency on `is_classifier()` method and `SKLearn` constants class from `raiutils`

### PR DESCRIPTION
**Pull request by gaugup**
## Description

Taking dependency on `is_classifier()` method and `SKLearn` method constants class from `raiutils`. This helps reduce some duplicate code in `raiwidgets` and `responsibleai`.
 
## Areas changed

npm packages changed:

- [ ] responsibleai/causality
- [ ] responsibleai/core-ui
- [ ] responsibleai/counterfactuals
- [ ] responsibleai/dataset-explorer
- [ ] responsibleai/fairness
- [ ] responsibleai/interpret
- [ ] responsibleai/localization
- [ ] responsibleai/mlchartlib
- [ ] responsibleai/model-assessment

Python packages changed:

- [x] raiwidgets
- [x] responsibleai
- [ ] erroranalysis
- [ ] rai_core_flask

## Tests

- [x] No new tests required.
- [ ] New tests for the added feature are part of this PR.
- [ ] I validated the changes manually.

## Documentation:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: raiutils/raiutils/models/models.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    """Provide scikit-learn related constants."""
+
+    EXAMPLES = 'examples'
+    LABELS = 'labels'
+    PREDICT = 'predict'
+    PREDICTIONS = 'predictions'
+    PREDICT_PROBA = 'predict_proba'
+
+
+def is_classifier(model):
```
Comment (imatiach-msft) [inline raiutils/raiutils/models/models.py:15]: I'm not completely sure, but you may need to add this to raiutils first, merge that PR, publish it, and then make a separate PR for raiwidgets to use this method

File: raiutils/raiutils/models/models.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    return (model is not None and
```
Comment (imatiach-msft) [inline raiutils/raiutils/models/models.py:16]: you may want to add some very simple test for this is_classifier method for code coverage

**Review by imatiach-msft — APPROVED** [review]

File: raiutils/raiutils/models/models.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation
```
Comment (imatiach-msft) [inline raiutils/raiutils/models/models.py:1]: what do you think about calling this file model_utils?  I always found it confusing to have the folder and file be the same name, eg models/models.  We could even break it into constants.py (or model_constants.py?) vs model_utils.py eventually, but probably it's too much to break it at this point, since there isn't a lot of code here yet.

File: responsibleai/responsibleai/utils.py [hunk 4]
```diff
@@ -1,10 +0,0 @@
-# Copyright (c) Microsoft Corporation
-# Licensed under the MIT License.
-
-from responsibleai._internal.constants import SKLearn
-
-
-def _is_classifier(model):
```
Comment (imatiach-msft) [inline responsibleai/responsibleai/utils.py:7]: nice to see the duplicate code removed!

**Review by romanlutz — APPROVED** [review]
I agree with @imatiach-msft 's comments. Thanks for adding this @gaugup !

**Review by raghavthind2005 — APPROVED** [review]

File: raiwidgets/requirements.txt [hunk 5]
```diff
@@ -8,3 +8,4 @@ scikit-learn>=0.22.1
 lightgbm>=2.0.11
 erroranalysis>=0.1.31
 fairlearn>=0.7.0
+raiutils>=0.0.1
```
[the remark this answers is not in the document]
Comment (romanlutz) [reply raiwidgets/requirements.txt:11]: Is it referenced directly, though? If it is then it still makes sense to mention it explicitly.

File: raiutils/raiutils/models/models.py (hunk 1 above)
Comment (gaugup) [reply raiutils/raiutils/models/models.py:15]: done.

File: raiutils/raiutils/models/models.py (hunk 2 above)
Comment (gaugup) [reply raiutils/raiutils/models/models.py:16]: These tests were written as a part of the PR https://github.com/microsoft/responsible-ai-toolbox/pull/1295

File: raiutils/raiutils/models/models.py (hunk 3 above)
Comment (gaugup) [reply raiutils/raiutils/models/models.py:1]: The suggested change was made under https://github.com/microsoft/responsible-ai-toolbox/pull/1295.

**Review by imatiach-msft — APPROVED** [review]

File: raiwidgets/requirements.txt (hunk 5 above)
[the remark this answers is not in the document]
Comment (gaugup) [reply raiwidgets/requirements.txt]: If we import methods from raiutils in raiwidgets, then it makes sense to take direct dependency on raiutils. @imatiach-msft  what do you think?

[the remark this answers is not in the document]
Comment (gaugup) [reply raiwidgets/requirements.txt]: Guessing it is ok to take depending on raiutils in raiwidgets. Resolving this comment.
